readme.md

```diff
@@ -341,50 +341,64 @@ Router.onRouteChangeError = (err, url) => {
   <ul><li><a href="./examples/with-prefetching">Prefetching</a></li></ul>
 </details></p>
 
-Next.js has an API which allows you to prefetch pages.
+Next.js exposes an API to prefetch pages.
 
-Since Next.js server-renders your pages, this allows all the future interaction paths of your app to be instant. Effectively Next.js gives you the great initial download performance of a _website_, with the ahead-of-time download capabilities of an _app_. [Read more](https://zeit.co/blog/next#anticipation-is-the-key-to-performance).
+> Since Next.js server-renders your pages, this allows all the future interaction paths of your app to be instant. Effectively Next.js gives you the great initial download performance of a _website_, with the ahead-of-time download capabilities of an _app_. [Read more](https://zeit.co/blog/next#anticipation-is-the-key-to-performance).
 
-> With prefetching Next.js only download JS code. When the page is getting rendered, you may need to wait for the data.
+:information_source: When prefetching, Next.js only downloads the code. Once the page is getting rendered you may still need to wait for data.
 
-#### With `<Link>`
+#### Using the `<Link>` Component
 
-You can add `prefetch` prop to any `<Link>` and Next.js will prefetch those pages in the background.
+To prefetch a route substitute your usage of [`<Link>`](#using-the-link-component) with the default export of `next/prefetch`. For example:
 
 ```jsx
-import Link from 'next/link'
+// components/navigation.js
+import Link from 'next/prefetch'
 
-// example header component
 export default () => (
   <nav>
     <ul>
-      <li><Link prefetch href='/'><a>Home</a></Link></li>
-      <li><Link prefetch href='/about'><a>About</a></Link></li>
-      <li><Link prefetch href='/contact'><a>Contact</a></Link></li>
+      <li><Link href='/'><a>Home</a></Link></li>
+      <li><Link href='/about'><a>About</a></Link></li>
+      <li><Link href='/contact'><a>Contact</a></Link></li>
     </ul>
   </nav>
 )
 ```
 
-#### Imperatively
+When this higher-level `<Link>` component is first used, the `ServiceWorker` gets installed.
```

Comment:
I pushed [an update](https://github.com/zeit/next.js/pull/1256/commits/48b454605733c60b759038720cab230332b3e527) to address this.